apps/meteor/app/api/server/v1/users.ts

```diff
@@ -555,6 +559,129 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'users.listByStatus',
+	{
+		authRequired: true,
+		validateParams: isUsersListStatusProps,
+		permissionsRequired: ['view-d-room', 'view-outside-room'],
+	},
+	{
+		async get() {
+			const { offset, count } = await getPaginationItems(this.queryParams);
+			const { sort, fields } = await this.parseJsonQuery();
+			const { status, roles, searchTerm } = this.queryParams;
+
+			const projection = {
+				name: 1,
+				username: 1,
+				emails: 1,
+				roles: 1,
+				status: 1,
+				active: 1,
+				avatarETag: 1,
+				lastLogin: 1,
+				type: 1,
+				reason: 0,
+				...fields,
+			};
+
+			const actualSort: Record<string, 1 | -1> = sort || { username: 1 };
+
+			if (sort?.status) {
+				actualSort.active = sort.status;
+			}
+
+			if (sort?.name) {
+				actualSort.nameInsensitive = sort.name;
+			}
+
+			let match: Filter<IUser>;
```

Comment:
```suggestion
			const match: Filter<IUser> = {};
```
We generally avoid using `let` :grimacing: Please change this to `const` and assign each property individually below (instead of setting the whole object).